### PR TITLE
[SIS-1533] Play 2.7 upgrade

### PR DESCRIPTION
**Pull request by cjrudd**

### CODE REVIEW

File: project/SilencerSettings.scala [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    libraryDependencies ++= Seq(
+      compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.7.0" cross CrossVersion.full),
+      "com.github.ghik" % "silencer-lib" % "1.7.0" % Provided cross CrossVersion.full
+    ),
+    // silence all warnings on autogenerated files
+    scalacOptions += "-P:silencer:pathFilters=target/.*",
+    // Make sure you only exclude warnings for the project directories, i.e. make builds reproducible
+    scalacOptions += s"-P:silencer:sourceRoots=${baseDirectory.value.getCanonicalPath}"
+  )
+}
```
Comment (fancellu) [inline project/SilencerSettings.scala:16]: NL needed

Comment (cjrudd) [reply project/SilencerSettings.scala]: fixed

**Review by fancellu — APPROVED** [review]
Fine

**Review by steve-edmans — APPROVED** [review]
LGTM

**Pull request merged**